### PR TITLE
feat: save and delete user testing profiles

### PR DESCRIPTION
**Pull request by Plow74**

### CODE REVIEW

File: src/components/direct/shared/Profile.tsx [hunk 1]
```diff
@@ -1,110 +1,284 @@
...
+    severity: 'info' | 'error' | 'success' | 'warning'
+  }
+
+  const [message, setMessage] = useState<Message>({ text: '', severity: 'info' })
+
+  useEffect(() => {
+    async function fetchLoggedInUsersProfiles() {
+      setIsLoading(true)
+      const loggedInUsersProfiles = await fetchProfiles()
+      const filteredProiles = removeProfilesWithNullProfileName(loggedInUsersProfiles)
```
Comment (drbgfc) [inline src/components/direct/shared/Profile.tsx:68]: Might want to fix typo: Proiles -> Profiles

File: src/components/direct/shared/Profile.tsx [hunk 2]
```diff
@@ -1,110 +1,284 @@
...
+      email: email,
+      username: username,
+      password: password,
+      istls: tls,
+      profilename: profilename,
+    }).then(async (response) => {
+      if (response) {
+        setMessage({ text: `${profilename} saved`, severity: 'success' })
+        const loggedInUsersProfiles = await fetchProfiles()
+        const filteredProiles = removeProfilesWithNullProfileName(loggedInUsersProfiles)
```
Comment (drbgfc) [inline src/components/direct/shared/Profile.tsx:101]: same

**Review by drbgfc — CHANGES_REQUESTED** [review]
This is awesome, man, thanks so much! Glad to see the reusable approach.
This is reviewed/approved but I'd like the build issue addressed before merging.
I didn't notice it, but, if your using the shared custom switch with label, after you pull, there will be an error due to a fixed typo. So, just fix the import if that's the case.

Once addressed, feel free to Squash and merge into dev yourself.

File: src/components/direct/shared/Profile.tsx (hunk 1 above)
Comment (Plow74) [reply src/components/direct/shared/Profile.tsx]: fixed

File: src/components/direct/shared/Profile.tsx (hunk 2 above)
Comment (Plow74) [reply src/components/direct/shared/Profile.tsx]: fixed

**Pull request merged**